doc/advanced.rst

```diff
@@ -83,11 +83,36 @@ Image scrapers are functions (or callable class instances) that do two things:
    extensions, respectively)
 3. Return rST that embeds these figures in the built documentation.
 
-The function should take the following inputs (in this order): ``block``,
-``block_vars``, and ``gallery_conf``. It should return a string containing the
-rST for embedding this figure in the documentation.
-See :func:`~sphinx_gallery.scrapers.matplotlib_scraper` for
```

Comment:
I think that it is a pity that the ref to the matplotlib scraper function disappeared. I typical find examples very useful to understand such an API.